umock-c does not work as it collides with sysroot headers (redefinition of several datatypes between local stdint.h and yocto provided stdint.h)
As mocking is only required for testing, we just deleted the provided versions in umock-c to use sysroot ones.
this prevents the following errors:
  In file included from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/recipe-sysroot/usr/include/sys/types.h:155,
                   from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/recipe-sysroot/usr/include/stdlib.h:394,
                   from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/umock_c.h:10,
                   from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/umocktypes.h:1
                   from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:10
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/recipe-sysroot/usr/include/bits/stdint-intn.h:27:19: error: conflicting types for 'int64_t'
     27 | typedef __int64_t int64_t;
        |                   ^~~~~~~
  In file included from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:6:
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/stdint.h:10:28: note: previo
     10 | typedef long long          int64_t;
        |                            ^~~~~~~
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c: In function 'umocktypes
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:189:43: error: format '%
    189 | IMPLEMENT_TYPE_HANDLERS(int64_t, int64_t, "%"PRId64)
        |                                           ^~~
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:27:50: note: in definiti
     27 |             size_t length = sprintf(temp_buffer, printf_specifier, *value); \
        |                                                  ^~~~~~~~~~~~~~~~
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:189:1: note: in expansio
    189 | IMPLEMENT_TYPE_HANDLERS(int64_t, int64_t, "%"PRId64)
        | ^~~~~~~~~~~~~~~~~~~~~~~
  In file included from /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/src/umocktypes_stdint.c:8:
  /build/build_imx8/tmp/work/aarch64-poky-linux/azure-identity-service/1.2.5-r0/git/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/inttypes.h:10:25: note: form
     10 | #define PRId64       "lld"
  cc1: all warnings being treated as errors


diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:31.039429985 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:31.039429985 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	2022-01-28 09:11:30.879425888 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/stdint.h	2022-01-28 09:11:30.879425888 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/deps/umock-c/inc/umock_c/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:31.339437663 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:31.339437663 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/c-shared/testtools/ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:31.855450867 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:31.855450867 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/azure-ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:32.451466120 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:32.451466120 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	2022-01-28 09:11:32.287461923 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/stdint.h	2022-01-28 09:11:32.287461923 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/deps/umock-c/inc/umock_c/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:32.759474002 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:32.759474002 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/c-utility/testtools/ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	2022-01-28 09:11:33.215485673 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	2022-01-28 09:11:33.215485673 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/deps/ctest/inc/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/inttypes.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/inttypes.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	2022-01-28 09:11:33.055481577 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/inttypes.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef INTTYPES_H
-#define INTTYPES_H
-
-#define PRId8        "hhd"
-#define PRId16       "hd"
-#define PRId32       "d"
-#define PRId64       "lld"
-
-#define PRIu8        "hhu"
-#define PRIu16       "hu"
-#define PRIu32       "u"
-#define PRIu64       "llu"
-
-#endif // INTTYPES_H
diff -aurN a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/stdint.h b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/stdint.h
--- a/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/stdint.h	2022-01-28 09:11:33.055481577 +0100
+++ b/tpm/aziot-tpm-sys/azure-iot-hsm-c/deps/utpm/deps/umock-c/inc/umock_c/aux_inc/stdint.h	1970-01-01 01:00:00.000000000 +0100
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-#ifndef STDINT_H
-#define STDINT_H
-
-typedef signed char        int8_t;
-typedef short              int16_t;
-typedef int                int32_t;
-typedef long long          int64_t;
-typedef unsigned char      uint8_t;
-typedef unsigned short     uint16_t;
-typedef unsigned int       uint32_t;
-typedef unsigned long long uint64_t;
-
-typedef signed char        int_least8_t;
-typedef short              int_least16_t;
-typedef int                int_least32_t;
-typedef long long          int_least64_t;
-typedef unsigned char      uint_least8_t;
-typedef unsigned short     uint_least16_t;
-typedef unsigned int       uint_least32_t;
-typedef unsigned long long uint_least64_t;
-
-typedef signed char        int_fast8_t;
-typedef int                int_fast16_t;
-typedef int                int_fast32_t;
-typedef long long          int_fast64_t;
-typedef unsigned char      uint_fast8_t;
-typedef unsigned int       uint_fast16_t;
-typedef unsigned int       uint_fast32_t;
-typedef unsigned long long uint_fast64_t;
-
-#define INT8_MIN         (-127i8 - 1)
-#define INT16_MIN        (-32767i16 - 1)
-#define INT32_MIN        (-2147483647i32 - 1)
-#define INT64_MIN        (-9223372036854775807i64 - 1)
-#define INT8_MAX         127i8
-#define INT16_MAX        32767i16
-#define INT32_MAX        2147483647i32
-#define INT64_MAX        9223372036854775807i64
-#define UINT8_MAX        0xffui8
-#define UINT16_MAX       0xffffui16
-#define UINT32_MAX       0xffffffffui32
-#define UINT64_MAX       0xffffffffffffffffui64
-
-#endif /* STDINT_H */
